docs/interacting-with-geth/rpc/ns-debug.md

```diff
@@ -685,17 +687,10 @@ hash.
 
 In addition to the hash of the transaction you may give it a secondary _optional_ argument, which specifies the options for this specific call. The possible options are:
 
-- `disableStorage`: `BOOL`. Setting this to true will disable storage capture (default = false).
```

Comment:
I move them into the file built-in tracer, as they are only available for the opcode tracer

Reply:
Right yeah I hate those options. Can you add them back but mark them somehow as only valid for the opcode tracer and ignored for the others? Because technically they are part of this TraceConfig object

Reply:
ok, so let's list them in the following table